app/config/factionhall/rooms.json

```diff
@@ -247,13 +249,17 @@
         },
         {
             "id": "medium-lounge",
-            "name": "Bayard Rustin Boudoir",
-            "type": "JITSI",
-            "muteRoom": false,
-            "hideVideo": false,
-            "capacity": 10,
-            "backgroundImage": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/pride/PrideRoomStudy.png",
-            "description": "I believe in social dislocation and creative trouble.",
+            "name": "McMartin Chamber",
+            "type": "CHATSTREAM",
+            "capacity": 666,
+            "doorTint": "#000000",
+            "muteRoom": true,
+            "hideVideo": true,
+            "backgroundImage": "https://tstvhq.s3.amazonaws.com/backgrounds/factionhall/greyfaction/gfmcmartinchamber.jpeg",
+            "iframeOptions": {
+            "src": "https://www.youtube.com/embed/_O0UgJgxI44"
+            "description": "“I am no longer whole / It's trauma based mind control.” - Satanic Planet
```

Comment:
"description": “I am no longer whole / It's trauma based mind control.” - Satanic Planet